test/connectors/neon.test.ts

```diff
@@ -0,0 +1,15 @@
+import { describe } from "vitest";
+import connector from "../../src/connectors/neon";
+import { testConnector } from "./_tests";
+
+describe.runIf(process.env.POSTGRESQL_URL)(
```

Comment:
I wish we could somehow mock `@neondatabase/serverless` (not sure if in their SDK they have it already). We could use something like pglite to emulate their interface in order to allow automated tests.